api/src/main/java/jakarta/data/Order.java

```diff
@@ -95,6 +95,15 @@ public static final <T> Order<T> by(Sort<T>... sorts) {
         return new Order<T>(List.of(sorts));
     }
 
+    /**
+     * The instances of {@link Sort} belonging to this {@code Order}.
+     *
+     * @return the instances of {@link Sort}, from highest precedence to lowest precedence.
+     */
+    public List<Sort<T>> getSorts() {
+        return sorts;
```

Comment:
Could you make this return immutable?

```suggestion
        return java.util.Collections.unmodifiableList(sorts);
```

Reply:
Ah, yes, of course, nice catch.